pydantic/types.py

```diff
@@ -2708,6 +2708,46 @@ def __get_pydantic_core_schema__(cls, source_type: Any, handler: GetCoreSchemaHa
         bool,
         None,
     ]
+    """A `JsonValue` is used to represent a value that can be serialized to JSON.
+
+    It may be one of:
+
+    * `List['JsonValue']`
+    * `Dict[str, 'JsonValue']`
+    * `str`
+    * `int`
+    * `float`
+    * `bool`
+    * `None`
```

Comment:
This whole bit looks pretty redundant in the docs:

![Screenshot from 2023-11-07 12-31-38](https://github.com/pydantic/pydantic/assets/3627481/911ce03e-1fd6-4f5d-b159-19525e32c935)

IMO this section can just be removed. Or to make it a bit more concise:

> A `JsonValue` is used to represent a value that can be serialized to JSON, i.e. a combination of lists, dicts, strings, numbers, booleans, and `None`.